ui/app/app.js

```diff
@@ -13,12 +13,12 @@ export default class App extends Application {
   podModulePrefix = config.podModulePrefix;
   Resolver = Resolver;
   engines = {
-    configUi: {
+    'config-ui': {
```

Comment:
i believe these needed to be changed to kebab case to match how they're mounted in the router: https://github.com/hashicorp/vault/blob/638f9ddae63894cf13f31b971bb496832cde918e/ui/app/router.js#L18

i haven't double checked this though